content/events/20-21/230_verkiezingen.md

```diff
@@ -31,6 +31,8 @@ Alvast een overzichtje van de huidige kandidaten:
 | Voorzitter & Vice | Francis & Arthur |
 | Penning           | Sammy            |
 | Sysadmin          | Jasper           |
+| 🦊		    | Ruben            |
+| <img src="https://steamcdn-a.akamaihd.net/steamcommunity/public/images/items/1195690/989d3b1f425b0ff2a196adecef360f561b20a187.png" width="35" height="35" /> | Maxime |
```

Comment:
We proberen niet naar images te hotlinken (om te voorkomen dat die uiteindelijk verdwijnen of veranderen), maar gebruiken daar ZeusWPI voor (https://zeus.ugent.be/zeuswpi/).